src/app/_components/Slider.js

```diff
@@ -0,0 +1,21 @@
+'use client';
+
+import PropTypes from 'prop-types';
+import { Swiper } from 'swiper/react';
```

Comment:
I would also reexport SwiperSlider named it Slide, so you can import everything from one place and later you can switch to another lib

Reply:
Reexported SwiperSlide